checkstyle.xml

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE module PUBLIC
+  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+  "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+	<property name="charset" value="UTF-8"/>
+	<property name="tabWidth" value="4"/>
+    <module name="TreeWalker">
+        <module name="AvoidStarImport">

```

Comment:
First of all I would be nice to group all modules into sections, where we can find them in checkstyle doku, so there would be much easier to find them or add more in the future. I meant somenthing following:
```suggestion
 <!-- Annotations-->
        <annotations-modules>
 <!-- Block Checks-->
        <block-checks modules>
```
etc. I hope you will get my point

Reply:
Ok, I will rearrange it.